gradio/mix.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, *interfaces: gradio.Interface, **options):
 
         async def parallel_fn(*args):
             return_values_with_durations = await asyncio.gather(
-                *[interface.call_function(0, args) for interface in interfaces]
+                *[interface.call_function(0, list(args)) for interface in interfaces]
```

Comment:
why recast as list, isn't it already a list? calling list is making a shallow copy for no reason?

Reply:
It's actually a tuple not a list :)